test/core-flags/cache-flags.test.js

```diff
@@ -136,6 +136,17 @@ describe('cache related flags from core', () => {
         expect(exitCode).toEqual(0);
     });
 
+    it('should assign cache build dependencies with default config', () => {
+        // TODO: Fix on windows
+        if (isWindows) return;
```

Comment:
Avoid this here

Reply:
already skipped for windows, some problems in all tests 😞

Reply:
Yep, let's focus on it late